build.gradle

```diff
@@ -644,6 +645,7 @@ project(':fmlonly') {
                           '--add-modules', 'ALL-MODULE-PATH',
                           // Additions to these JVM module args should be mirrored to server_files/args.txt and other similar blocks in the buildscript
                           '--add-opens', 'java.base/java.util.jar=cpw.mods.securejarhandler',
+                          '--add-opens', 'java.base/java.lang.invoke=cpw.mods.securejarhandler',
```

Comment:
As stated above this line shouldn't this be mirrored to the [server_files/args.txt](https://github.com/MinecraftForge/MinecraftForge/blob/1.19.x/server_files/args.txt)?

Also seems to be missing from the [LauncherJson](https://github.com/MinecraftForge/MinecraftForge/blob/1.19.x/build.gradle#L646) task